src/constants/coin-config.js

```diff
@@ -108,12 +109,21 @@ export default {
         { label: 'Mainnet', value: 'prod' },
         { label: 'Testnet (Kovan)', value: 'test' }
       ]
-    }
+    },
+    trx: {
+      fullName: 'Tron',
+      supportedRecoveries: [],
+      icon: trxIcon,
+      envOptions: [
+        { label: 'Mainnet', value: 'prod' },
+        { label: 'Testnet', value: 'test' }
```

Comment:
Should this be `Testnet (Tronex)` like in ethereum?

Reply:
oh yeah, good idea. ok